src/cli/config/config.go

```diff
@@ -89,11 +91,15 @@ func InitFunc(cmd *cobra.Command, args []string) error {
 		fp = configFilePath
 	}
 
-	err := initWithViper(GetViper(cmd.Context()), fp)
+	vpr := GetViper(cmd.Context())
+
+	err := initWithViper(vpr, fp)
 	if err != nil {
 		return err
 	}
 
+	SetViper(cmd.Context(), vpr)
```

Comment:
bug here: note the return value of SetViper.  `context.Context` is not mutable.  if you don't overwrite the local contxt value with the one returned here, you'll lose the viper embedding.

Proper handling would be to set `ctx := cmd.Context()` at the top of the func, and then:
```suggestion
	ctx = SetViper(ctx, vpr)
```